htdocs/langs/fr_FR/boxes.lang

```diff
@@ -72,6 +72,11 @@ BoxProposalsPerMonth=Proposition par mois
 NoTooLowStockProducts=Pas de produits sous le seuil de stock minimal
 BoxProductDistribution=Répartition des produis/services
 BoxProductDistributionFor=Répartition des %s pour les %s
+BoxTitleLastModifiedSupplierBills=
```

Comment:
If you want to have key on transifex, you must add entries in en_US files. And you must defined the value for en_US, so translator will know what is english value to translate.